c2corg_ui/templates/helpers/view.html

```diff
@@ -200,6 +177,25 @@ <h4 translate>No maps associated</h4>
   % endif
 </%def>
 
+<%def name="delete_association_rights(document)">\
```

Comment:
This function name is not consistent: it does not delete the association rights. By the way it could be used for something else then removing associations (perhaps for adding associations?).
``test_association_rights``?